src/commands/send/receive.command.ts

```diff
@@ -145,6 +145,8 @@ export class SendReceiveCommand extends DownloadCommand {
                 } else {
                     return Response.error(e);
                 }
+            } else {
+                return Response.error(e);
             }
```

Comment:
I was thinking something like:
```suggestion
                }
            }
            return Response.error(e);
```
but might not be possible?

Reply:
Yeah, I do like that, just a single `return Response.error(e);` statement at the end of the `catch {}` block vs. the added `else` statements in there (since it's either going to `return` early and exit the block, or fall down ultimately to the final line of the block, `return Response.error(e);`. Simplifies things a little bit

Reply:
oic, yeah that is better, thanks!